sass/base/_base.scss

```diff
@@ -0,0 +1,6 @@
+html{
+    position: absolute;

```

Comment:
Not sure why you'd need this to be absolutely positioned? It's not normally recommended for the entire HTML document to be absolutely positioned